server/routes/forecast.ts

```diff
@@ -735,7 +735,9 @@ export default class ForecastService {
         this.client
       );
       const response = await callWithRequest('forecast.searchForecaster', {
-        body: {},
+        body: {
+          size: MAX_FORECASTER,
```

Comment:
This is from the setting for # of forecasters we allow in plugin?

Reply:
this is hardcoded in frontend, not a setting call to backend. Will need to fix it in the future.